### ISSUE TITLE
Discuss TEI-encoded tables and how to balance accuracy with a11y requirements

### ISSUE THREAD

**Issue by erinchambers** [CDRH/data#263]
TEI-encoded tables are varied in their purpose, layout, and headers (or lack thereof) depending on the source material. Tables need to have certain elements and/or attributes to meet WCAG 2.1 standards, too. Laura, Karin, and Erin Kay should meet at some point to discuss when/how to remediate existing TEI tables when we work on them, and if/how we should encode tables differently (or update XSLT) going forward.

**Comment by erinchambers** [comment]
Just a quick note that Lexus Root, DH summer fellow & Cather Archive grad assistant, is also interested in this discussion & would like to know what we decide.
